app/components/Form/ProjectMilestoneReportForm.tsx

```diff
@@ -16,7 +16,7 @@ import {
   paymentSchema,
   paymentUiSchema,
 } from "data/jsonSchemaForm/paymentSchema";
-import { ProjectMilestoneReportForm_reportingReqiurement$key } from "__generated__/ProjectMilestoneReportForm_reportingReqiurement.graphql";
+import { ProjectMilestoneReportForm_reportingRequirement$key } from "__generated__/ProjectMilestoneReportForm_reportingRequirement.graphql";
```

Comment:
Do you know why this is showing as diffs? This file looks exactly the same to me